src/wurf/http_resolver.py

```diff
@@ -51,7 +51,8 @@ def resolve(self):
             filename = None
 
         file_path = self.url_download.download(cwd=folder_path,
-            source=self.source, filename=filename)
+                                               source=self.source,
+                                               filename=filename)
```

Comment:
This is a good example where the pep8-style vertical alignment looks extremely ugly. It is like a toothbrush!
There are lots of examples like this after the reformatting.
@mortenvp and I consistently used the single indentation throughout this project. That looks much better, although it is explicitly forbidden in http://legacy.python.org/dev/peps/pep-0008/#indentation
I think that this kind of vertical alignment should be reverted.